### ISSUE TITLE
Is there any way to combine language models?

### ISSUE THREAD

**Issue by harinath141** [tmbdev/ocropy#169]
## Steps to Reproduce (for bugs)

1.
2.
3.
4.

## Your Environment

* Python version:

* Git revision of ocropy: 
* Operating System and version:

**Comment by harinath141** [comment]
Also I'm getting error "index 60 is out of bounds for axis with size 60" when I try to load existing model and train.

**Comment by harinath141** [comment]
Any one kindly guide me i am using Telugu uni code Its showing fine in ground truth text file but showing the image as above..

**Comment by zuphilip** [comment]
Please provide more information: explain step-by-step and in details such that it is reproducible for anyone else. Don't ignore all points from the issue template, cf. https://github.com/tmbdev/ocropy/issues/168.

Moreover, it looks for me that you have two questions about training and one question about line generation, which may be independent of each other.

Please copy the exact commands you use here and copy the exact error message with all intermediate messages here. Maybe someone can help you then.

**Comment by harinath141** [comment]
@zuphilip Thanks for your response.

1. I am trying to train Ocropus in telugu (unicode language) while training I have a text file containing Telugu text Im trying to generate ground truth and bin.png images using ocropus-linegen using command
ocropus-linegen -t tests/sri.txt -f tests/mallanna.ttf
The images generated is showing consonant seperately and Vowel Seperate in Telugu this both together will give correct charecter,In gt its giving exact word as desired .plz find the above image and text for ref.

2.I have trained Telugu with images and gt txt and I set a saving frequency of 200 iterations so i have about 15-16 models of telugu .Will i have to take last iteration saved model or is there any way to combine all the models?

3.When i try to train on the top of telugu language model i previously generated using --load option
Its throwing error "index 60 is out of bounds for axis with size 60" 
I am using the command "ocropus-rtrain --load tel.pyrnn.gz -o tel-plus *.bin.png -c *.gt.txt -f 200 -S200"

**Comment by harinath141** [comment]
The problem point 3 is due to addition of more training data than previously trained data

**Comment by zuphilip** [comment]
2.) Normally, it is expected that the last model you obtain is the best one, because it is trained the most. However, you should evaluate your models on some evaluation data. Then you will have for each model an error rate and you can stop if the error is not changing or you are happy with the current error rate. You can for example look at https://github.com/jze/ocropus-model_oesterreich-ungarn for a similar example.

It seems for me that 15*200 = 3000 iterations might be not enough to train a model, especially if it is a completely new language with (a lot of?) different characters.

**Comment by zuphilip** [comment]
3.) Shouldn't you provide also the character set with `-c` as discussed in the other issue? If the error still exists, then please show all lines of the error message as well as your command you used.
